shared/js/background/events.es6.js

```diff
@@ -183,75 +183,79 @@ browser.webRequest.onBeforeRequest.addListener(
     additionalOptions
 )
 
-if (manifestVersion === 2) {
-    const extraInfoSpec = ['blocking', 'responseHeaders']
-    if (browser.webRequest.OnHeadersReceivedOptions.EXTRA_HEADERS) {
-        extraInfoSpec.push(browser.webRequest.OnHeadersReceivedOptions.EXTRA_HEADERS)
-    }
+const extraInfoSpec = manifestVersion === 3 ? ['responseHeaders'] : ['blocking', 'responseHeaders']
```

Comment:
Nit: Doesn't really matter, but my preference would be to do something like:
```suggestion
const extraInfoSpec = ['responseHeaders']
if (manifestVersion === 2) {
    extraInfoSpec.push('blocking')
}
```

Might be nice to add a comment too, something to explain that for MV2 webRequest is doing the blocking but we still need the event listener on MV3 to help do the reporting stuff.

Reply:
Could we make this and the indenting change a separate PR; largely I don't think much of this file has changed but reviewing it is a little harder than it should be ideally.

Reply:
Scrap my indenting comment, it's easy to filter on the cli and in github options (the patch looks really great without whitspace now :) ).